src/main/webapp/app/exercises/quiz/manage/statistics/short-answer-question-statistic/short-answer-question-statistic.component.ts

```diff
@@ -118,19 +118,4 @@ export class ShortAnswerQuestionStatisticComponent extends QuestionStatisticComp
         });
         this.updateData();
     }
-
-    /**
-     * Get the solution that was mapped to the given spot in the sample solution
-     *
-     * @param spot {object} the spot that the solution should be mapped to
-     * @return the mapped solution or undefined if no solution has been mapped to this location
-     */
-    correctSolutionForSpot(spot: ShortAnswerSpot) {
```

Comment:
This removes the only usage of `ShortAnswerSpot` which causes an "defined but never used" warning on line 11 now. (=> Remove the import.)

Reply:
Done :)